test/integration/util/util.go

```diff
@@ -80,6 +81,11 @@ func runCommand(command string, commandPath string) (stdOut string, stdErr strin
 	return
 }
 
+func WaitForNSeconds(seconds int) error {
```

Comment:
Why wrapping this up in a named function when it is just a sleep, especially since there is only one usecase for this.